### PR TITLE
Initial Issue Triage Guide

### PR DESCRIPTION
**Pull request by lmbr-pip**
Add a small but useful triage guide

* Encode CSVS mapping to issue priority

### CODE REVIEW

File: TRIAGE_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+# Overview
+
+SIG-Security holds weekly triage on [Mondays](https://lists.o3de.org/g/o3de-calendar/calendar). High priority issues can be triaged via Discord as and when required.
+
+The standard O3DE [triage guide](https://github.com/o3de/community/) should be used to cover process for accepting issues and setting standard labels.
```
Comment (allisaurus) [inline TRIAGE_GUIDE.md:6]: there does not seem to be a triage guide at this link?

Comment (allisaurus) [inline TRIAGE_GUIDE.md:6]: there does not seem to be a triage guide at this link?

File: TRIAGE_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    * Remove the `needs-triage` label and add `triage-accepted` label
+    * Set a priority
+* Or assign a reviewer/maintainer to reproduce or get more information.
+
+## Useful Links
+* Issues to triage for SIG: https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-triage+label%3Asig%2Fsecurity
+* Known security issues: https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fsecurity
+
+## SIG Specific Guide
+* Ensure issues have label `kind\security` set on them so SIG security can find issues shared wit other SIGs.
```
Comment (allisaurus) [inline TRIAGE_GUIDE.md:19]: nit: "with" other SIGs

Comment (allisaurus) [inline TRIAGE_GUIDE.md:19]: nit: "with" other SIGs

File: TRIAGE_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* SIG-Security only owns issues for code SIG-Security (see [charter] for areas of ownership) or actively intends to work on.
```
Comment (allisaurus) [inline TRIAGE_GUIDE.md:20]: not sure if empty [charter] link intentional, I know that doc is undergoing revisions.

Comment (allisaurus) [inline TRIAGE_GUIDE.md:20]: not sure if empty [charter] link intentional, I know that doc is undergoing revisions.

**Review by allisaurus — APPROVED** [review]
LG, sans minor comments

File: TRIAGE_GUIDE.md (hunk 1 above)
Comment (lmbr-pip) [reply TRIAGE_GUIDE.md:6]: The intent is for community to own the common triage guide, so deferring to that. Have reached out privately to contributors who were adding this, also created general issue: https://github.com/o3de/community/issues/130

**Review by amzn-changml — APPROVED** [review]
Aside from what @allisaurus pointed out (and it seems resolved), this looks good to me. 

I am interested in hearing more about any 3rd party SIR selection process from the TSC, but it's likely outside of scope for this guide.

**Pull request merged**